worker/caasrbacmapper/mapper.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package caasrbacmapper
+
+import (
+	"sync"
+	"time"
+
+	"github.com/juju/errors"
+	"gopkg.in/juju/worker.v1"
+	"gopkg.in/juju/worker.v1/catacomb"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	core "k8s.io/client-go/informers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+
+	"github.com/juju/juju/caas/kubernetes/provider"
+)
+
+// Mapper describes an interface for mapping k8s service account UID's to juju
+// application names.
+type Mapper interface {
```

Comment:
is this used outside the package?
let's not export unless needed

Reply:
Yeah it is.